src/views/components/infographic-banner.html

```diff
@@ -1,5 +1,5 @@
 <div class="c-infographic-banner">
-	<img src="{{@shared.assetsHost}}/img/landing/infographic-bg.png" srcset="{{@shared.assetsHost}}/img/landing/infographic-bg@2x.png 2x">
+	<img src="{{@shared.assetsHost}}/img/landing/infographic-bg.png" srcset="{{@shared.assetsHost}}/img/landing/infographic-bg@2x.png 2x" loading="lazy">
```

Comment:
For anything above the fold I dont think there is a point to do lazy loading. Here the whole point of the page is the image, so lazyloading would make a bad user experience. Please remove it

Reply:
Done, let me know if there are any other images you want me to revert :)